webknossos/webknossos/dataset/dataset.py

```diff
@@ -441,6 +505,178 @@ def download(
                 exist_ok=exist_ok,
             )
 
+    @classmethod
+    def from_images(
+        cls,
+        input_path: Union[str, PathLike],
+        output_path: Union[str, PathLike],
+        voxel_size: Optional[Tuple[float, float, float]] = None,
```

Comment:
I did not expect voxel_size to be optional, how does the Dataset class handle this?

Reply:
Good point, I think this was only possible in the constructor for backwards-compatibility, when opening existing datasets. Will change here.